packages/cli-poa/src/util/log.js

```diff
@@ -0,0 +1,5 @@
+import utils from '@percy/sdk-utils';
```

Comment:
remove this file and use `import logger from '@percy/logger';` directly wherever you're consuming

Reply:
Reverting back to `utils` as tests are breaking since few modules use CommonJS and few use ESModules, which is causing issues while doing named imports.